src/misc_functions/returnhelper.ts

```diff
@@ -134,7 +146,9 @@ export class ReturnHelper<Errorkind extends BlankCommandError = CommandError> {
     public getShared(): ReturnData<Errorkind> {
         return this.data;
     }
-
+    public hasErrors(): boolean {
```

Comment:
This isn't really a pattern I want to encourage. The more warnings we can give (where appropriate) the better. But I haven't seen the uses of it yet, so maybe I'll be surprised.

Reply:
This is for the NBT parsing. I need to find what the correct parser is to use, so I have to see if the parser for the tag returned an error (and hence is incorrect)

Reply:
But errors can just be warning. The pattern I have been following is use fail for fatal parsing error, and succeed when there is only warnings/invalid data.